test/fixtures/indices.yml

```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  finalized_at: 2016-01-13 16:11:26
+  category_id: :one
```

Comment:
All of these were added by the migration, is this something I need to check every time?